### PR TITLE
Port l10n_eu_nace to v9

### PR DESCRIPTION
**Pull request by lbelorgey**

### CODE REVIEW

File: l10n_eu_nace/partner.py [hunk 1]
```diff
@@ -19,22 +19,22 @@
...
 
 
-class PartnerCategory(osv.Model):
+class PartnerCategory(models.Model):
     """Let users search on code without a dot"""
     _inherit = 'res.partner.category'
 
-    def name_search(self, cr, uid, name, args=None, operator='ilike',
-                    context=None, limit=80):
+    def _name_search(self, cr, user, name='', args=None, operator='ilike',
```
Comment (pedrobaeza) [inline l10n_eu_nace/partner.py]: As you are already passing this to new API, change the signature removing cr and user, and put @api.model decorator.

Comment (lbelorgey) [inline l10n_eu_nace/partner.py]: Done. Thanks.

File: l10n_eu_nace/partner.py [hunk 2]
```diff
@@ -19,22 +19,21 @@
...
 
-    def name_search(self, cr, uid, name, args=None, operator='ilike',
-                    context=None, limit=80):
+    @api.model
+    def name_search(self, name, args=None, operator='ilike', limit=100):
         """When no results are found, try again with an additional "."."""
-        results = super(PartnerCategory, self).name_search(
-            cr, uid, name, args=args, operator=operator, context=context,
-            limit=limit)
+        results = super(PartnerCategory, self).name_search(name, args=args,
```
Comment (pedrobaeza) [inline l10n_eu_nace/partner.py]: This is not PEP8 valid. Please put all arguments in the following line with one indentation more.

Comment (lbelorgey) [inline l10n_eu_nace/partner.py]: Sorry. Forgot to run flake8. Thanks.

**Comment by pedrobaeza** [comment]
:+1:

**Comment by gurneyalex** [comment]
:+1:

**Pull request merged**